components/map/index.tsx

```
@@ -0,0 +1,69 @@
+} from '@vis.gl/react-google-maps';
+import { useCallback } from 'react';
+
+interface MapProps {
+  location: {
+    latitude: number;
+    longitude: number;
+  };
+  onLocationChange: (location: { latitude: number; longitude: number }) => void;
+}
+
+const Map: React.FC<MapProps> = ({
```

Biome

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)